www/source/docs/blueprints.md

```diff
@@ -0,0 +1,58 @@
+# Blueprints
+Blueprints are default implementations of the build and runtime phases for your application. They also provide some default health check hooks where appropriate to ensure your application functioning reliably. Customized Blueprints can be created to facilitate reusability of common patterns in your organization for developing, building, and running your applications.
+
+## Getting Started
+### New Projects
+Simply run `hab plan init` to leverage blueprints. Habitat will look in your application code for known project types and the appropriate `pkg_blueprint`.
+
+If you would like to specify the desired blueprint to use, you can run `hab plan init java8-maven-tomcat`
```

Comment:
Given our current implementation of `hab plan init`, `java8-maven-tomcat` would map to `PKG_NAME`. My guess is this would be an option (i.e. `--blueprint java8-maven-tomcat` or similar) but I don't think it's critical for this doc.